guide/standards.md

```diff
@@ -0,0 +1,51 @@
+# Standards
+
+The TELUS Design System maintains a set of standards and practices for core components and community components.
+
+## What is Core and Community?
+
+**TDS Core** provides a library of highly-reusable components and foundational principles that can be composed and combined to help solve
+complex designs rapidly. **TDS Community** is a centralized library of components that are not quite foundational, but are
+identified as reusable among several teams at TELUS.
+
```

Comment:
Should there be also a note that Community Component can potentially become a Core Component if it becomes highly-reusable and foundational in future?

Reply:
We'll address that in a later PR.